packages/google_maps_flutter/google_maps_flutter_ios/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: google_maps_flutter_ios
 description: iOS implementation of the google_maps_flutter plugin.
 repository: https://github.com/flutter/packages/tree/main/packages/google_maps_flutter/google_maps_flutter_ios
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+maps%22
-version: 2.15.2
+version: 2.15.3
```

Comment:
Similarly, the only change to this package is to a file in the example app that's not the one visible on pub.dev, so this should be exempt from versioning.

Reply:
You're right, not sure how that ended up getting removed or at what point. I think the reasonable thing to do would be to make ios's `PlatformMarker` expect an integral zIndex, so that this truncation would now be a type error to which we can pass `zIndexInt`

Reply:
That's seems reasonable from flags.
`-Wno-float-conversion` looks like it allows implicit float conversion which is what was happening
`-Wshorten-64-to-32` would disallow the long to int cast that's happening at the line you pointed out